R/pkg/R/utils.R

```diff
@@ -440,19 +446,23 @@ processClosure <- function(node, oldEnv, defVars, checkedFuncs, newEnv) {
         # Namespaces other than "SparkR" will not be searched.
         if (!isNamespace(func.env) ||
               (getNamespaceName(func.env) == "SparkR" &&
-              !(nodeChar %in% getNamespaceExports("SparkR")))) {  # Only include SparkR internals.
+              !(nodeChar %in% getNamespaceExports("SparkR")))) {
```

Comment:
According to the style guide (http://google-styleguide.googlecode.com/svn/trunk/Rguide.xml#indentation),  we should align this line with `!isNamespace` right?

Maybe, we can see the same issue in various places and should open another PR about it.